CMakeLists.txt

```diff
@@ -399,17 +405,20 @@ if(BUILD_TESTING)
     endforeach()
 endif()
 
-foreach(examplename ${openPMD_EXAMPLE_NAMES})
-    if(${examplename} MATCHES ".+parallel$")
-        if(openPMD_HAVE_MPI)
+# Current examples all use HDF5, elaborate if other backends are used
+if(openPMD_HAVE_HDF5)
```

Comment:
don't exclude the build, just the test below is fine.
the examples should still be build and only throw errors at runtime if they try to use a backend that does not exist.

Reply:
I included the examples as tests in CI. With this approach, CI tests without HDF5 will always fail.

Reply:
no, the `add_tests` are below and properly guarded which are the runtime. but the compile should still be done.